fuel-vm/src/checked_transaction.rs

```diff
@@ -110,27 +110,14 @@ impl<Tx: IntoChecked> Checked<Tx> {
     }
 }
 
-impl<Tx: IntoChecked + ExecutableTransaction + fuel_tx::field::GasLimit> Checked<Tx>
+// #[cfg(feature = "internals")]
```

Comment:
Should we just remove these instead of commenting out?

Reply:
I do it after this PR is merged so it doesn't undo the approvals